### PR TITLE
Install JaCoCo

### PR DESCRIPTION
**Pull request by RamOwen99**
To see how many code lines are cover with unit testing, JaCoCo will provide us a report if our code lines have their unit testing, also we will use JaCoCo to determine what will be the percentage of unit testing that we need to cover.
To see more details go to https://trello.com/c/wqCwIDRh/77-us-35-nfr-implement-jacoco-on-at-scre-api

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -116,6 +123,46 @@
                             <appName>at-sce-api-qa</appName>
                         </configuration>
                     </plugin>
+                    <!-- JaCoCo -->
```
Comment (foji9315) [inline pom.xml:126]: you are missing important exclusions in jacoco's configuration

**Review by foji9315 — CHANGES_REQUESTED** [review]

**Review by luisgalindoluna — APPROVED** [review]
Looks good to me. Approved.

File: pom.xml (hunk 1 above)
Comment (ivanflores15) [reply pom.xml:126]: you forgot add classes/packages such as domain, constants, config and exceptions

**Review by ivanflores15 — CHANGES_REQUESTED** [review]

**Review by ivanflores15 — APPROVED** [review]
good job

**Review by EmiliaGuillen-dev — APPROVED** [review]
looks great

**Pull request merged**